### PR TITLE
Fix/bead ids bug134

### PR DESCRIPTION
**Pull request by lisatiberio**
This fixes the issues demonstrated by the new bead file.

### CODE REVIEW

**Comment by jakebeal** [comment]
Tests are failing --- can you please check into the reasons?

**Comment by lisatiberio** [comment]
Jake,
The unit test, test_colormodel is failing when I run make test-matlab or make test-octave.  However, when I run test_colormodel using the Matlab interactive environment, my new changes fix the bug.  Have you seen this behavior before?

Thanks,
Lisa

**Comment by jakebeal** [comment]
That sounds like contamination from one test to another through TASBEConfig.

... and I think I see the problem in test_beadpeaks:

    TASBEConfig.set('override_autofluorescence',true);
If that's true when test_colormodel runs, that will definitely mess it up.

There are two fixes for this:
1. A kludge that we should do for now, which is to set it back to false after the CM in the test resolves.
2. Implement some sort of save-state functionality on TASBEConfig (maybe a "push" and "pop"?) so that we can rewind after contaminating it.  I have been thinking about this for some time, and will open an issue.

File: tests/test_beadpeaks.m [hunk 1]
```diff
@@ -92,3 +92,51 @@
...
+% TASBEConfig.set('override_autofluorescence',true);
+% 
+% function test_rightpeaks
+% 
+% [CM] = setupBV421CM();
+% % Execute and save the model
+% CM=resolve(CM);
+% 
+% % Reset TASBEConfig to not contaminate other tests.
+% TASBEConfig.set('override_autofluorescence',[]);
```
Comment (jakebeal) [inline tests/test_beadpeaks.m]: should be `false`, not empty; I believe that's the problem here.

File: tests/test_colormodel.m [hunk 2]
```diff
@@ -62,6 +62,7 @@
 CM=set_translation_channel_min(CM,[2,2,2]);
 
 TASBEConfig.set('path', '/tmp/plots');
+TASBEConfig.set('override_autofluorescence',[]);
```
Comment (jakebeal) [inline tests/test_colormodel.m]: Please remove this line - it's too much kludge.  We should be cleaning up in the prior file, not catching in the one that happens to be next in the current (fragile) order.

**Comment by jakebeal** [comment]
This pull is just waiting on resolution of #133, since it includes the OutputSettings changes as well.

**Comment by jakebeal** [comment]
Fixes #134

**Pull request merged**